content/src/main/content/jcr_root/apps/acs-commons/authoring/sitewide-component-props/sitewide-component-props.js

```diff
@@ -0,0 +1,55 @@
+(function ($, ns, channel, window, undefined) {
+
+    var actionDef = {
+        icon: 'coral-Icon--globe',
+        text: Granite.I18n.get('Configure Shared Content'),
```

Comment:
Should change this comment to Configure Site-wide Component Properties to be consistent with the rest of our naming